### ISSUE TITLE
Error in CreateMotifObject function of TF Footprinting Vignette

### ISSUE THREAD

**Issue by strawberry789** [timoast/signac#331]
motif <- CreateMotifObject(
  positions = motif.positions,
  pwm = pwm
)

The above command gives an error message:
Error in eval(expr, envir, enclos): lazy-load database '.../Signac.rdb' is corrupt

I tried re-installing Signac, but the same error message pops up.  Thank you.

**Comment by timoast** [comment]
Try restarting R and clearing the environment

**Comment by strawberry789** [comment]
Thanks!  It did work!

But then I continued with the vignette and I encountered another error.

Here is the command I entered:

brain <- Footprint(
  object = brain,
  motif.name = c("list of gene names"),
  genome = BSgenome.Mmusculus.UCSC.mm10
)

And I get the following error:

Error in GetFootprintRegions(motif.obj = motif.obj, motif.name = x): Motif not found

What is the criteria for motif.name?  Do these need to be transcription factors?  Or can be any genes?

Thank you.
